packages/client/hmi-client/src/components/workflow/util.ts

```diff
@@ -5,12 +5,26 @@ export const enum RunType {
 	Ciemss = 'ciemss'
 }
 
-export const getTimespan = (inputTimespan: TimeSpan, dataset?: CsvAsset) => {
+export type DatasetType = {
+	data: { x: number; y: number }[];
+	label: string;
+	fill: boolean;
+	borderColor?: string;
+	borderWidth?: number;
+	borderDash?: number[];
+};
+
+export const getTimespan = (inputTimespan: TimeSpan, dataset?: CsvAsset): TimeSpan => {
 	let start = inputTimespan.start;
 	let end = inputTimespan.end;
 	// If we have the min/max timestamp available from the csv asset use it
+	// Try to get `timestamp` first, then `t`
+	// TODO: figure out a more robust way to get the timestamp column from the csv asset
 	if (dataset) {
-		const tIndex = dataset.headers.indexOf('timestamp');
+		let tIndex = dataset.headers.indexOf('timestamp');
+		if (tIndex === -1) {
+			tIndex = dataset.headers.indexOf('t');
+		}
```

Comment:
You should have and leverage a mappings data structure provided to you that tells you which column is the timestamp column.

Reply:
Done!